code_challenges.rb

```diff
@@ -21,6 +32,18 @@
 album3 = 'Abbey Road'
 # Expected output: 'bby Rd'
 
+def no_vowels(string)
+  no_vowels.delete('aeiouAEIOU')
+end
+
+p no_vowels(album1)
+p no_vowels(album2)
+p no_vowels(album3)
+
+ # // the above method does not work. I receive an error message.
+ # // the simple code below works, however it's not set up as a true method that can be run repeatedly with a new argument each time...
+
```

Comment:
Ah got it, so something to keep in mind (and I have seen this happen in your JS challenges in the past too (not trying to rip on you, just giving a heads up)). 

Generally speaking we don't (can't?) call on the method or function name inside the function or method itself. 

so here on line 36 you are going inside of the method called **_no_vowels_** and there, inside the method, you are calling on **_no_vowels_**. 

**_no_vowels_** is being defined by you the developer so it doesn't know what to do with **_no_vowels_** inside its own definition. 

Now there are times to call on a function inside of itself: thats called recursion. We don't do a whole lot of recursion in class as its a concept more present in Functional Programming.